### ISSUE TITLE
Make the help dialog more "helpful"

### ISSUE THREAD

**Issue by mrmr1993** [philc/vimium#1280]
To quote myself from [my comment here](https://github.com/philc/vimium/issues/1269#issuecomment-65218294), I think we should

> ... push all but the basic commands out of the help dialog and onto a separate page:

> * we stop needing a scrollable modal window (which is bad UX);
* there's no pressure to reduce the number of commands;
* there's space to discuss quirks (eg. we can't activate hover styles for `LinkHints.activateModeToHover` in #1032, we don't detect elements with click handlers for `LinkHints.activate`, etc.);
* users are more likely to search (searching in a modal dialog is weird);
* users searches for commands don't interact with the underlying page (eg. try opening help and searching for "scroll" on this page);
* the help dialog would be a place where (mainly new) users go to "get help" with using the extension (instead of being our sole user-facing documentation for commands).

In short: we should compile some proper documentation for commands, and then use the help dialog to reveal the most basic commands or direct users to the full documentation.

**Comment by dragon788** [comment]
I have to say I like the idea of more help pages (tabs in the modal experience?) and I'm with mrmr1993 in wanting a keyboard exclusive interface for Chrome. As soon as I get some time to play with tiling window managers on Windows I'll probably stop touching my mouse almost entirely throughout the day.
